stdlib/multiprocessing/reduction.pyi

```diff
@@ -44,9 +44,9 @@ if sys.platform == "win32":
 
 else:
     if sys.platform == "darwin":
-        ACKNOWLEDGE: Literal[True]
+        ACKNOWLEDGE: Final = True
     else:
-        ACKNOWLEDGE: Literal[False]
+        ACKNOWLEDGE: Final = False
```

Comment:
It might be better just to do ACKNOWLEDGE: Final[bool]` here, without the `sys.platform` check. I worry that the existing stubs might be encoding some platform-specific implementation details here that users probably shouldn't rely on.

Reply:
https://github.com/python/typeshed/pull/12332/commits/44a969fa649f92b2008ac9c08ace879c4f4682ee